Tests/SentryTests/Integrations/Performance/UIViewController/SentryTimeToDisplayTrackerTest.swift

```diff
@@ -16,15 +16,17 @@ class SentryTimeToDisplayTrackerTest: XCTestCase {
 
         init() {
             framesTracker = SentryFramesTracker(displayLinkWrapper: displayLinkWrapper)
+            SentryDependencyContainer.sharedInstance().framesTracker = framesTracker
+            print("Fixture.init.SentryDependencyContainer.sharedInstance().framesTracker = \(SentryDependencyContainer.sharedInstance().framesTracker)")
             framesTracker.start()
         }
 
         func getSut(for controller: UIViewController, waitForFullDisplay: Bool) -> SentryTimeToDisplayTracker {
-            return SentryTimeToDisplayTracker(for: controller, framesTracker: framesTracker, waitForFullDisplay: waitForFullDisplay)
+            return SentryTimeToDisplayTracker(for: controller, waitForFullDisplay: waitForFullDisplay)
         }
     }
 
-    private let fixture = Fixture()
+    private lazy var fixture = Fixture()
```

Comment:
It turns out that with a constant instance property, it was initialized as part of a pre-test process, and then when the tests ran, the dependency container shared instance was constructed again and had a difference instance of the frames tracker, so the tests would fail. Making it lazy defers initialization until the test process itself is running.